### ISSUE TITLE
Clear Glass Panes?

### ISSUE THREAD

**Issue by robertmaxton42** [SlimeKnights/TinkersConstruct#2992]
Are there any plans to re-introduce clear panes? Thought I'd ask before starting a build - they'd be perfect for what I have in mind.

**Comment by bonii-xx** [comment]
Currently no plans. I think we couldn't port them over straight and it takes too much effort to do properly. Ender-IO might have some

**Comment by KnightMiner** [comment]
Mostly they are just weird since its not a standard block shape, but it might be possible to just give it vertical connected states and use the pane shape to handle horizontal. Not the most trivial thing (especially since multipart cannot set default textures), but it could be a somewhat clean implementation in the end

**Comment by bonii-xx** [comment]
No, it wont. It's highly nontrivial because there are so many more cases than in any other situation. Regular connected blocks are bad enough already.

**Comment by Raghalin** [comment]
TConstruct-1.12-2.8.1.49
Forge version: 1.12.2 - 14.23.1.2555

multiplayer:
#1 when pouring from smeltery, if you look away then look back. the white box frame for the glass appears and you can see the blue coloring filling while the block is in the cooling stage.

#2 if standing to close to where block is placed, the clear glass is 'used' (counter drops by 1) but no clear glass is actually placed. aka you could place a 64 stack into 1 block and still not have a block apear.

sorry if this isn't enough info, I'm clueless about coding.
also running 
buildcraft-7.99.12
ImmersiveEngineering-0.12-75

**Comment by KnightMiner** [comment]
@Raghalin please make a separate issue report, those are unrelated to clear glass being implemented or not.

**Comment by KnightMiner** [comment]
Added in 1.16, no plans to add to 1.12.
